src/objectExplorer/objectExplorerService.ts

```diff
@@ -191,10 +195,13 @@ export class ObjectExplorerService {
         }
     }
 
-    public updateNode(node: TreeNodeInfo): void {
+    public updateNode(node): void {
```

Comment:
Please always have types (I would suggest making a type union of the possible types a node could be)